packages/framework/resources/views/layouts/scripts.blade.php

```diff
@@ -1,5 +1,7 @@
 {{-- The compiled Vite scripts --}}
-@if(Asset::exists('app.js'))
+@if(Vite::running())
```

Comment:
May make more sense to have this alongside the CSS include since the script is deferred anyways, but not sure how to handle that best with the partials since it would fit better in the `head` component, and don't want to make such a big change here.